### PR TITLE
[Doc] Update field

### PR DESCRIPTION
**Pull request by FantasyVR**
Related issue = #

### CODE REVIEW

File: docs/lang/articles/basic/field.md [hunk 1]
```diff
@@ -57,14 +69,19 @@ a `3 x 2` matrix. To allocate a `128 x 64` matrix field which has a
 `3 x 2` matrix for each of its entry, use the statement
```
Comment (strongoier) [inline docs/lang/articles/basic/field.md:69]: Make lines 67-69 become
````
In a general case, suppose you have a `128 x 64` field called `A`, and each element is
a `3 x 2` matrix, you can define it with
````

File: docs/lang/articles/basic/field.md [hunk 2]
```diff
@@ -57,14 +69,19 @@ a `3 x 2` matrix. To allocate a `128 x 64` matrix field which has a
...
 `A = ti.Matrix.field(3, 2, dtype=ti.f32, shape=(128, 64))`.
 
+### Access matrix field elements
 - If you want to get the matrix of grid node `i, j`, please use
   `mat = A[i, j]`. `mat` is simply a `3 x 2` matrix.
 - To get the element on the first row and second column of that
   matrix, use `mat[0, 1]` or `A[i, j][0, 1]`.
+- The 0-D matrix fields `x = ti.Matrix.field(n=3, m=4, dtype=ti.f32, shape=())` should be accessed by `x[None][p, q]`, (`0<= p < n, 0<= q < m`).
```
Comment (strongoier) [inline docs/lang/articles/basic/field.md:77]:
```suggestion
- The 0-D matrix field `x = ti.Matrix.field(n=3, m=4, dtype=ti.f32, shape=())` should be accessed by `x[None][p, q]` (`0 <= p < n, 0 <= q < m`).
```

File: docs/lang/articles/basic/field.md [hunk 3]
```diff
@@ -57,14 +69,19 @@ a `3 x 2` matrix. To allocate a `128 x 64` matrix field which has a
...
+
+:::note
 - As you may have noticed, there are **two** indexing operators `[]`
   when you load a matrix element from a global matrix field: the
```
Comment (strongoier) [inline docs/lang/articles/basic/field.md:81]:
```suggestion
  when you access a member of a matrix from a matrix field: the
```

File: docs/lang/articles/basic/field.md [hunk 4]
```diff
@@ -57,14 +69,19 @@ a `3 x 2` matrix. To allocate a `128 x 64` matrix field which has a
...
   first is for field indexing, the second for matrix indexing.
```
Comment (strongoier) [inline docs/lang/articles/basic/field.md:82]:
```suggestion
  first is for field indexing, and the second is for matrix indexing.
```

File: docs/lang/articles/basic/field.md [hunk 5]
```diff
@@ -106,6 +123,7 @@ Members of a struct field can be accessed either locally (i.e., member of a stru
 # set the position of the first particle to origin
 particle_field[0] # local ti.Struct
 particle_field[0].pos = ti.Vector([0.0, 0.0, 0.0])
+particle_field[0].pos[0] = 1.0
```
Comment (strongoier) [inline docs/lang/articles/basic/field.md:126]:
```suggestion
```
This doesn't match the comment in L123.

Comment (FantasyVR) [reply docs/lang/articles/basic/field.md]: Thanks for your review!

**Review by strongoier — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**